fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/ColumnValidator.java

```diff
@@ -122,6 +122,7 @@ public void validateSqlInjection(String schema, String... conditions) {
         return tableColumnMap;
     }
 
+    @SuppressWarnings("StringSplitter")
```

Comment:
Why this you suppress this?

Reply:
Using the Splitter implementation in this method causes an errors in some integration test, I looked into but wasnt making any inroads to find out what was causing this. Will create an issue on issue tracker to remove the annotation.